helpers/aws.js

```diff
@@ -271,19 +271,26 @@ class AwsClient {
     /**
      * Delete s3 object
      * @method removeObject
-     * @param {String}              object       object name
-     * @param {Function}            callback        callback function
+     * @param {String}              cacheKey       Path to cache
      */
-    removeObject(object, callback) {
+    removeObject(cacheKey) {
```

Comment:
Is an argument name of this function correct?
It seems not a "cache" key.

Reply:
Changed to objectKey.